src/App.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import './App.css';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import Navbar from './components/Navbar';
+import BookList from './components/BookList';
+import Categories from './components/Categories';
+import FormAddBook from './components/FormAddBook';
+
+function App() {
+  return (
+    <BrowserRouter>
+      <Navbar />
+      <Routes>
+        <Route path="/" element={<BookList />} />
+        <Route path="/categories" element={<Categories />} />
+      </Routes>
+      <FormAddBook />
```

Comment:
- [ ] Add book form is also displaying on the categories page(route). Please make sure it's only displayed on the books page. To fix this call add book form component inside the booklist.